### ISSUE TITLE
Markers: AttributeError when calling __repr__ / __str__

### ISSUE THREAD

**Issue by elabrosseRodeofx** [PixarAnimationStudios/OpenTimelineIO#592]
The `otio.schema.Marker` object's `__repr__` and `__str__` methods are referencing a 'media_reference' and a 'source_range' attribute.
however Markers do not have those attributes. They do however have a 'marked_range' attribute that is not referenced in the `__repr__` / `__str__`

Printing a Marker instance results in an AttributeError:
```
>>> otio.schema.Marker()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/devLocal/elabrosse/rdoenv/auto_lineup/otio/lib/python2.7/site-packages/opentimelineio/schema/marker.py", line 25, in __repr__
    repr(self.media_reference),
AttributeError: 'opentimelineio._otio.Marker' object has no attribute 'media_reference'
```

From what I see it seems that the content of opentimelineio/schema/clip.py was copy/pasted almost as is in opentimelineio/schema/marker.py.

My corporation is waiting for confirmation that our CLA was received. We will submit a PR as soon as possible.

**Comment by jminor** [comment]
Yep, we got the CLA. Thanks for finding this oversight.

**Comment by elabrosseRodeofx** [comment]
Made a PR:

https://github.com/PixarAnimationStudios/OpenTimelineIO/pull/596

**Comment by reinecke** [comment]
I accidentally resolved this in #740 after it'd already been resolved in #596 (Oops!)
